### PR TITLE
Added project euler problem13 

### PR DESCRIPTION
**Pull request by parulagg27**
**Fixes issue:**

**Changes:**

Added readme file and python code for project euler problem 13.

### CODE REVIEW

File: code/online_challenges/src/project_euler/problem_013/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Project Euler Problem #13: Large Sum
+
+([Problem Link](https://projecteuler.net/problem=13))
+
+Work out the first ten digits of the sum of the following one-hundred 50-digit numbers.
+
+<p align = "center">
+37107287533902102798797998220837590246510135740250
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_013/README.md]: I would just put this in code formatting (It might be shorter + easier to read)

Comment (parulagg27) [reply code/online_challenges/src/project_euler/problem_013/README.md]: I did thought of doing the same. But what about center alignment of these numbers in original question, I need to maintain? Should I add spaces for each number? Please guide @arnavb :thinking:

Comment (arnavb) [reply code/online_challenges/src/project_euler/problem_013/README.md]: @parulagg27 Don't worry about the center alignment; It's readability that matters the most!

Comment (parulagg27) [reply code/online_challenges/src/project_euler/problem_013/README.md]: Completely agree. :+1:

**Comment by parulagg27** [comment]
Made required changes. Please review @arnavb.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**
positive_opengenus

**Pull request merged**